### PR TITLE
Refactor subcommand arguments; add simple dataset import subcommand

### PR DESCRIPTION
**Pull request by davecap**

### CODE REVIEW

File: solvebio/cli/importers.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                    print("No template with ID {0} found!"
+                          .format(args.template_id))
+            else:
+                tpl = None
+                print("Cannot create a new dataset: "
+                      "no template ID specified by the --template-id flag!")
+
+            if not tpl:
+                print("Choose an ID from one of the following templates: ")
+                print(solvebio.DatasetTemplate.all())
```
Comment (jsh2134) [inline solvebio/cli/importers.py]: should this query be limited in some way?

Comment (davecap) [inline solvebio/cli/importers.py]: we only have 1 template right now, but yea maybe in the future...

File: solvebio/cli/importers.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+            genome_builds = [args.genome_build] if args.genome_build else None
+            dataset = solvebio.Dataset.get_or_create_from_full_name(
+                full_name=args.dataset,
+                genome_builds=genome_builds,
+                is_genomic=tpl.is_genomic,
+                entity_type=tpl.entity_type,
+                fields=tpl.fields)
+
+            if not dataset:
+                sys.exit(1)
```
Comment (jsh2134) [inline solvebio/cli/importers.py]: add explanatory error message here?

**Comment by jsh2134** [comment]
👍

**Pull request merged**